app/controllers/api/v1/visualization/annotations_controller.rb

```diff
@@ -0,0 +1,180 @@
+module Api
+  module V1
+    module Visualization
+      # API methods for visualizing annotation-related data
+      # does NOT contain methods for editing annotations
+      class AnnotationsController < ApiBaseController
+        include Concerns::Authenticator
+        include Concerns::StudyAware
+        include Concerns::ApiCaching
+        include Swagger::Blocks
+
+        VALID_SCOPE_VALUES = ['study', 'cluster']
+        VALID_TYPE_VALUES = ['group', 'numeric']
+
+        before_action :set_current_api_user!
+        before_action :set_study
+        before_action :check_study_view_permission
+        before_action :check_api_cache!
+        after_action :write_api_cache!
+
+        annotation_description_doc = 'Object with name (String), values (Array or uique values), type (String), scope (String), and cluster_name (string, if applicable)'
+
+        swagger_path '/studies/{accession}/annotations' do
+          operation :get do
+            key :tags, [
+                'Visualization'
+            ]
+            key :summary, 'Get all annotations for the study'
+            key :description, 'Get all annotations for the study, with name, values, type, scope, and cluster_name if applicable'
+            key :operationId, 'study_annotations_path'
+            parameter({
```

Comment:
ah, that's a remnant of when I had these factored out in a different way.  I found that calling the method with an object, rather than a block, made it easier to pull out shared params between methods.  Agree that since that's not what's happening here, block-passing matches rest of the codebase better.